components/inputs/input-time-range.js

```diff
@@ -98,6 +160,17 @@ class InputTimeRange extends RtlMixin(LocalizeCoreElement(LitElement)) {
 		if (!this.label) {
 			console.warn('d2l-input-time-range component requires label text');
 		}
+
+		if (!this.startValue) {
+			const startValue = getDefaultTime();
+			this.startValue = formatTimeInISO({hours: startValue.getHours(), minutes: startValue.getMinutes(), seconds: startValue.getSeconds()});
+		}
+		if (!this.endValue) {
+			const endValue = getDateFromISOTime(this.startValue);
+			endValue.setMinutes(endValue.getMinutes() + getIntervalNumber(this.timeInterval));
+			this.endValue = formatTimeInISO({hours: endValue.getHours(), minutes: endValue.getMinutes(), seconds: endValue.getSeconds()});
+		}
```

Comment:
Is there a reason we can't put these in the constructor?

Reply:
The assignment of `startValue` could but `endValue` depends on if `startValue` has a value so I think we'd want to keep that here.